.github/workflows/go.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         fetch-depth: 0
     - name: Commit linting
       if: matrix.os == 'ubuntu-latest' && github.ref != 'refs/heads/master'
-      uses: wagoid/commitlint-github-action@v5
+      uses: wagoid/commitlint-github-action@v2.0.2
```

Comment:
it seems the latest is `https://github.com/wagoid/commitlint-github-action/releases/tag/v2.0.3`
any reason to stick to the previous release?

Reply:
This command is failing, so I was trying to see if previous/other version would work. Apparently they aren't working either :(